File: API/Models/Saida.cs
using System;

namespace API.Models
{
    public class Saida
    {
        public Saida()
        {
            HoraSaida = DateTime.Now;
        }
        public int SaidaId { get; set; }
        public int PessoaId { get; set; }  
        public Pessoa Pessoa { get; set; }  
        public int CarroId { get; set; } 
        public Carro Carro { get; set; }  
        public DateTime HoraEntrada { get; set; } 
        public DateTime HoraSaida { get; set; } 
         
    }
}

File: FRONT/src/app/models/saida.ts
import { Pessoa } from 'src/app/models/pessoa';
import { Carro } from 'src/app/models/carro';
export interface Saida { 
    saidaId? : number;
    pessoaId : number;
    pessoa? : Pessoa;
    carroId : number;
    carro? : Carro;
    dinheiro? : number;
    horaEntrada? : Date;
    horaSaida?: string;
}